reference/configuration/framework.rst

```diff
@@ -2359,11 +2359,6 @@ collect_serializer_data
 When this option is ``true``, all normalizers and encoders are
 decorated by traceable implementations that collect profiling information about them.
 
-.. deprecated:: 7.3
-
-    Setting the ``collect_serializer_data`` option to ``false`` is deprecated
```

Comment:
I wonder if we should simply remove this option description as it doesn't have any affect anymore.